service.go

```diff
@@ -118,7 +121,7 @@ func GenerateServiceMonitor(s *v1.Service) *monitoringv1.ServiceMonitor {
 	}
 }
 
-// GenerateRoute generates an OpenShift route object based on the passed Service object.
+// Create route to the specified service.
```

Comment:
The general convention is to start all comments about a function with that function name.

 i.e.
```
//Foo is a function that does x
Foo() {
}

//Bar is a function does y
Bar() {

}
```